airlock/templates/file_browser/request/request.html

```diff
@@ -155,6 +155,19 @@
     {% #description_item title="Supporting files not for release" %}
       {{ release_request.supporting_files_count }}
     {% /description_item %}
+    {% #description_item title="Files released and uploaded" %}
+      <div class="flex flex-row flex-wrap gap-6" id="uploaded-files-count">
+        {{ release_request.uploaded_files_count }}
+        {% if release_request.status.value == "RELEASED" %}
+          {% icon_check_circle_solid class="h-5 w-5 text-green-700" %}
+        {% elif release_request.upload_in_progress %}
+          <span class="upload-in-progress"/>
```

Comment:
Is this span needed, or is it leftover from an earlier attempt?

Reply:
Ah, no it's not, will remove it

Reply:
Done :)